CareKitStore/CareKitStore/CoreData/OCKStore+Patients.swift

```diff
@@ -60,15 +60,12 @@ extension OCKStore {
                           completion: ((Result<[OCKPatient], OCKStoreError>) -> Void)? = nil) {
         context.perform {
             do {
-                try self.validateNumberOfPatients()
```

Comment:
Can we add this line back in? We'd like to tackle removing the 1 patient limit as a separate issue. That probably needs to be paired together with some other considerations, like updating the `OCKDailyTasksPageViewController` to make sure it show the tasks for a 1 specific patient.

Reply:
Not a problem, I added this back and reverted back to original test cases for patient

Reply:
Thanks! We'll get to removing the patient cap eventually!